packages/next/src/server/app-render/required-scripts.tsx

```diff
@@ -5,7 +5,7 @@ import ReactDOM from 'react-dom'
 export function getRequiredScripts(
   buildManifest: BuildManifest,
   assetPrefix: string,
-  crossOrigin: string | undefined,
+  crossOrigin: undefined | '' | 'anonymous' | 'use-credentials',
```

Comment:
In the future, we might want to change this from explicit type to some shared type that matches the caller